CONTRIBUTING.md

```diff
@@ -298,3 +298,37 @@ You can use any of the other common git trailers. In particular, you can use
 ```markdown
 Cc: @octocat
 ```
+
+### Acknowledging contributors
+
+_GLASS_ uses [All Contributors](https://allcontributors.org) to acknowledge
+different types of contributions to the project. The contributor data is stored
+in `.all-contributorsrc` as JSON and is rendered as Markdown in
+`docs/CONTRIBUTORS.md`. The markdown file is further rendered in
+[_GLASS_' documentation](https://glass.readthedocs.io/stable/#contributors).
```

Comment:
This link won't work yet, but I like the idea of it being `stable`. Do we leave it as it is or change to `latest`?

Reply:
Oh yes, `latest` does not sound bad here, given that only developers will read CONTRIBUTING.md

```suggestion
[_GLASS_' documentation](https://glass.readthedocs.io/latest/#contributors).
```